wallet-new/test/WalletNewGen.hs

```diff
@@ -0,0 +1,51 @@
+module WalletNewGen
```

Comment:
It's not a big deal, but I'm not sure I like calling this `WalletNewGen` since all of our other modules containing Hedgehog generators throughout the project have just been called `Gen`.

I'm wondering if we should have a `Test.Cardano.Wallet.Gen` or something.

Reply:
Yeah, I think following the standards of other packages would be good. `Test.Pos.Ssc.Gen` for example, or `Test.Pos.Crypto.Gen`. This should probably be `Test.Pos.Wallet.Gen`.

Reply:
I will make a separate PR and talk to the wallet-new team. I tried making the change but GHC was complaining.